src/App.js

```diff
@@ -54,12 +54,15 @@ class App extends Component {
   }
 
   render() {
+    const errorModal = this.state.error ? <ErrorModal message={this.state.error} displayHomePage={this.displayHomePage}/> : null
```

Comment:
Thank you for taking care of this. I agree, it makes the return statement much easier to read!